packages/libllvm/build.sh

```diff
@@ -93,6 +96,18 @@ termux_step_post_make_install () {
 	for tool in clang clang++ cc c++ cpp gcc g++ ${TERMUX_HOST_PLATFORM}-{clang,clang++,gcc,g++,cpp}; do
 		ln -f -s clang-${_PKG_MAJOR_VERSION} $tool
 	done
+
+	local OPENMP_ARCH
+	if [ $TERMUX_ARCH = "i686" ] || [ $TERMUX_ARCH = "x86_64" ]; then
+		OPENMP_ARCH="i386"
```

Comment:
The Termux clang x86-64 produces x86-64 code.

As I understand it one uses `LLVM_TARGET_ARCH=X86` for both x86 and x86-64, but llvm then looks at `LLVM_DEFAULT_TARGET_TRIPLE` to see if 32-bit or 64-bit code should be built.